src/views/helpers/IndexHelpers.elm

```diff
@@ -0,0 +1,14 @@
+module IndexHelpers exposing (..)
```

Comment:
What is `IndexHelpers`?

Reply:
They're functions that help support the view it's named after. The `Index.elm` file is being treated as an `HTML` only file. So it only has Html literals. I don't want to distract the view from the overall structure of the webpage.

Reply:
I am always wary of "helpers" or "utility" because they are a catch-all and can become meaningless.